behat-tests/features/bootstrap/FeatureContext.php

```diff
@@ -400,12 +424,14 @@ public function assertLinkVisibleRegion($link, $region) {
   }
 
   /**
-   * @Then I should not see the link :link visible in the :region( region)
+   * @Then I should not visibly see the link :link in the :region( region)
```

Comment:
That's not what I meant.... sorry let me try again.
The step is one that is defined in our own FeatureContext.  The question is why the change to the syntax?  Trying to find if there is a standard of syntax perhaps that would have the "visibly" before "see the link" rather than "see the link.... visible".